std/base64.d

```diff
@@ -391,7 +391,7 @@ template Base64Impl(char Map62th, char Map63th, char Padding = '=')
      *  The number of times the output range's `put` method was invoked.
      */
     size_t encode(E, R)(scope const(E)[] source, auto ref R range)
-    if (is(E : ubyte) && isOutputRange!(R, char))
+    if (is(E : ubyte) && isOutputRange!(R, char) && !is(R == char[]))
```

Comment:
The other overload had this check, so I'm just thinking it was forgotten but the compiler didn't complain because R was never an output range of char.

Reply:
>but the compiler didn't complain because R was never an output range of char

I specifically asked it to be removed IIRC in a different PR for this reason